### PR TITLE
Update hue_sensors.py

### PR DESCRIPTION
**Pull request by rossdargan**
Added in support for the hue tap.

Nice work on the component :)

### CODE REVIEW

File: custom_components/sensor/hue_sensors.py [hunk 1]
```diff
@@ -17,8 +17,9 @@
 _LOGGER = logging.getLogger(__name__)
 PHUE_CONFIG_FILE = 'phue.conf'
 SCAN_INTERVAL = timedelta(seconds=1)
-
-
+#... I preffer this naming, but happy to match yours
```
Comment (robmarkcole) [inline custom_components/sensor/hue_sensors.py]: Thanks for the contribution :-) 
Could you run the style checking with tox as per https://home-assistant.io/developers/development_testing/ 
Certainly tox will complain about the formatting of these comments.
Once tox is happy then can merge.
Cheers!

Comment (robmarkcole) [reply custom_components/sensor/hue_sensors.py]: Actually don't worry, tox only required for submission to HA which I'm handling in a different repo

**Comment by robmarkcole** [comment]
@rossdargan Checkout https://github.com/home-assistant/home-assistant/pull/9449

**Comment by rossdargan** [comment]
Cool. I'll take a look tonight!

It probably is worth trying to split out into a hub app. That way we could add the binary sensor I've worked on (it makes automation slightly easier).

**Comment by robmarkcole** [comment]
Agree about the Hue hub.
Can you explain how it is easier in automations?
Cheers

**Comment by rossdargan** [comment]
@robmarkcole it means that you have a binary sensor per button. It means you don't have to do string comparisons in the automation. To be honest it's not significantly easier and you do end up with a lot of sensors for a dimmer control... 

I've added an issue to collate thoughts around the hub (if you don't mind me jumping in after you have done the hard work)

**Comment by robmarkcole** [comment]
@rossdargan could you upload an example of the json returned for the TAP?
I don't have a TAP but will need to add it to the tests

**Pull request merged**